lib/squib/layout_parser.rb

```diff
@@ -20,6 +26,9 @@ def load_layout(files, initial = {})
         if File.exists? thefile
           # note: YAML.load_file returns false on empty file
           yml = layout.merge(YAML.load_file(thefile) || {})
+          if !@dpi_force and yml.key? 'deck' and yml['deck'].key? 'dpi'
+            @dpi = yml['deck']['dpi'].to_f
```

Comment:
I need to make copious tests for this. What's supposed to happen here is that if anyone is crazy enough to load multiple layouts with a `deck` key, each layout will be interpreted in the dpi specified by its own `deck`. Alternately we could just ignore anyone that crazy and call that undefined behavior.